### PR TITLE
Add's The ability to qep

### PR DESCRIPTION
**Pull request by LadyFowls**
Essentialy as the title suggests

### CODE REVIEW

**Comment by LadyFowls** [comment]
!review

**Comment by NonQueueingMatt** [comment]
this is why global warming exists

File: code/modules/mob/living/carbon/human/emote.dm [hunk 1]
```diff
@@ -556,6 +556,25 @@
 				else
 					message = "hugs [get_visible_gender() == MALE ? "himself" : get_visible_gender() == FEMALE ? "herself" : "themselves"]."
 
+		if("dab", "dabs")
```
Comment (MarinaGryphon) [inline code/modules/mob/living/carbon/human/emote.dm]: Dabbing doesn't exist in 2462. Consider changing it to be the famous Skrellian dance move, the qep. It's quite similar, actually.

Comment (LadyFowls) [reply code/modules/mob/living/carbon/human/emote.dm]: Fixed

File: code/modules/mob/living/carbon/human/emote.dm [hunk 2]
```diff
@@ -571,9 +571,9 @@
 					var/mob/living/carbon/human/H
 					if(ishuman(src))
 						H = src
-					message = "<span class='danger'>dabs on [M].</span>"
+					message = "<span class='danger'>qeps on [M].</span>"
 				else
-					message = "dabs on themself"
+					message = "qeps on themself"
```
Comment (OneOneThreeEight) [inline code/modules/mob/living/carbon/human/emote.dm]: qeps on themselves.*

**Comment by Arrow768** [comment]
Travis sais no because of compile warnings

**Pull request closed without merging**